src/app/core/components/content-top-bar/content-top-bar.component.ts

```diff
@@ -44,6 +47,27 @@ export class ContentTopBarComponent {
     filter(navigationNeighbors => !!navigationNeighbors?.isReady),
   );
 
+  allowWatchingGroup$ = this.currentContent$.pipe(
```

Comment:
I suppose most of what you have added here is repeated from the other place where this button appears... so there is a lot of code repeated. Probably it would have been wise to place that in a new component used at 2 places.

Reply:
Please, look the last commit:

I haven't found the way how current content could be used in page navigator, I suppose when we will move current content into for user page into `UserInfo` the page navigator will use the same approach as `GroupInfo` and then params of page navigator could be clarified